core/src/main/scala/zio/config/Config.scala

```diff
@@ -161,7 +161,7 @@ object Config {
    *
    * {{{
    *    final case class kafkaConfig(server: String, serde: String)
-   *    nested("KAFKA")(string("SERVERS") |@| string("SERDE"))(KafkaConfig.apply, KafkaConfig.unapply)
+   *    nested("KAFKA")(string.at("SERVERS") |@| string.at("SERDE"))(KafkaConfig.apply, KafkaConfig.unapply)
```

Comment:
All these examples should survive with no changes to syntax.

Reply:
All docs reverted.